tlsutil/config.go

```diff
@@ -561,6 +569,12 @@ func (c *Configurator) VerifyServerHostname() bool {
 	return c.base.VerifyServerHostname || c.autoEncrypt.verifyServerHostname
 }
 
+// TransportCredentials returns credentials to be used with eg a grpc server.
+func (c *Configurator) TransportCredentials() *credentials.TransportCredentials {
+	creds := credentials.NewTLS(&tls.Config{Certificates: []tls.Certificate{*c.Cert()}})
```

Comment:
`*c.Cert()` will panic if `c.Cert()` returns nil.

Currently the only call to `TransportCredentials` does a nil check before the function call, but I'm concerned about someone else calling this later without knowing about that behavior.

Maybe it would be useful to put in a nil check in this function just in case.

Alternatively, constructing the `TransportCredentials` could be done in the `GRPCServer` function, rather than being split out into a separate method.

Reply:
Nice catch! There is a nil check now.